hub/management/commands/import_council_control.py

```diff
@@ -0,0 +1,95 @@
+from datetime import date
+
+from django.conf import settings
+
+import pandas as pd
+
+from hub.import_utils import filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+party_shades = {
```

Comment:
This’ll be the third import script that defines an identical `party_shades` dict. I wonder whether we should put into into a shared file (maybe even `.base_importers`?) that we can then import in each script that needs it?